Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -1763,22 +1763,22 @@ namespace Tensile
         // Architecture dependent.
         else if(pAMDGPU->skDynamicGrid == 3)
         {
-            dim3 dSize(1, 1, 1);
+            size_t x, y, z = 1;
```

Comment:
Shouldn't this be like this?
  x=1, y=1,z=1
Are x and y really initialized?

Reply:
My bad, it should be.